extensions/amp-access/0.1/amp-access.js

```diff
@@ -464,16 +464,22 @@ export class AccessService {
    */
   applyAuthorizationToElement_(element, response) {
     const expr = element.getAttribute('amp-access');
-    const on = this.evaluator_.evaluate(expr, response);
-    let renderPromise = null;
-    if (on) {
-      renderPromise = this.renderTemplates_(element, response);
-    }
-    if (renderPromise) {
-      return renderPromise.then(() =>
-        this.applyAuthorizationAttrs_(element, on)
-      );
+    let on = false;
+    try {
+      on = this.evaluator_.evaluate(expr, response);
```

Comment:
Is only `evaluate()` expected to throw an error here? If so, the try/catch should be as narrow as possible.

Reply:
Sure. I'll make it more narrow + add a unit test.

Reply:
Good idea